include/swift/AST/DiagnosticsSema.def

```diff
@@ -1167,6 +1167,14 @@ WARNING(implementation_only_requires_library_evolution,none,
         "using '@_implementationOnly' without enabling library evolution "
         "for %0 may lead to instability during execution",
         (Identifier))
+WARNING(implementation_only_deprecated_explicit,none,
+        "'@_implementationOnly' is deprecated, use 'internal import' "
+        "and family instead",
```

Comment:
I think this could just be
```suggestion
        "'@_implementationOnly' is deprecated, use 'internal import' instead"
```
I'm not sure it would be clear what family is referring to.

Reply:
Makes sense, I applied it.